test/connectSpec.js

```diff
@@ -283,3 +481,17 @@ function * assertResolve (promise, expected) {
   assert(promise instanceof Promise)
   assert.deepEqual(yield promise, expected)
 }
+
+function * assertRejectsWith (promise, errorConstructor, errorMessage) {
```

Comment:
Addressed in https://github.com/interledger/js-ilp-plugin-bells/pull/52/commits/24328b6e34c5f570f1b0c224861b8fcf3ac15e14